src/utils/helper.ts

```diff
@@ -25,7 +23,9 @@ export const formatQuerySortList = (sorter: SorterResult<any> | SorterResult<any
   );
 
   const r = sorters.map((sorter) => ({
-    field: (sorter.field?.toString()! || sorter.columnKey?.toString()!).replaceAll('__', '.'),
+    field: (
+      sorter.field?.toString()!.replaceAll(',', '.') || sorter.columnKey?.toString()!
```

Comment:
why using `!` while using `or` operand and `?` ?

Reply:
Hmm I'm not sure I only added the ```.replaceAll(',', '.')``` part to handle array dataIndexes: ```dataIndex: ['study', 'study_code'],``` gives: ```'study,study_code'``` so I change it to ```study.study_code``` so we send the correct field key to the backend for sorting.

But I agree it makes no sense to have ```?```, ```!```and ```||```all at the same time, I just didn't want to modify the existing logic. But I could take a look across the different tables with sorting and see what's going on